Development.md

```diff
@@ -18,9 +18,9 @@ Remember to `git pull` every time you push a change.
 
 We use some private python libraries for our integrations therefore you need to configure artifactory as pypi repository:
 ```bash
-$ export ARTIFACTORY_URL=artifactory.stackstate.io/artifactory/api/pypi/pypi-local && \
-  export ARTIFACTORY_USER=... && \
-  export ARTIFACTORY_PASSWORD=... && \
+$ export ARTIFACTORY_PYPI_URL=artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-local/simple && \
+  export artifactory_user=... && \
+  export artifactory_password=... && \
```

Comment:
Why make the variable names lowercase?

Reply:
this is from StackVista group CI variables